conference/models.py

```diff
@@ -200,7 +201,7 @@ class DeadlineContent(models.Model):
     """
     Content of a deadline.
     """
-    deadline = models.ForeignKey(Deadline)
+    deadline = models.ForeignKey(Deadline, on_delete=models.CASCADE)
```

Comment:
btw, I think we can drop the Deadline (as a model) – we no longer use it anywhere for new conferences. But will need to double check that.